lib/pump.c

```diff
@@ -0,0 +1,132 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+//Read Data and convert into Struct
+const int NumSensors = 5;
+
+const int A =20;
+const int B =15;
+const int C =10;
+const int D =5;
+
+
+ struct DataStruct
+  {
+      int id;
+      int data;
+  } Datos;
```

Comment:
spanish? :)

Reply:
also, you can use for example `typedef struct { ... } pump_data`, then later on you can use just `pump_data` when you use this type (instead of `struct pump_data`)

Reply:
Regardless of language, I'd argue that naming a data structure "DataStruct" and a variable "Data" is not a great idea.